packages/ui/src/components/form-primitives/form-caption.tsx

```diff
@@ -37,7 +46,10 @@ export const FormCaption = ({ theme = 'default', className, children }: PropsWit
   const effectiveIconName = theme === 'danger' ? 'cross-circle' : 'warning-triangle-outline'
 
   return (
-    <p className={cn(formCaptionVariants({ theme }), className)}>
+    <p
+      className={cn(formCaptionVariants({ theme }), { 'cn-caption-disabled': disabled }, className)}
+      aria-disabled={disabled}
```

Comment:
Not sure on the need to use `aria-disabled`. Screen readers won't see this tag as having anything to do with the input anyway, so it'll just be seen as a disabled paragraph of text, which isn't really a thing. Typically this is something that's used with interactive elements, not static text.